tests/test_oml/test_registry/test_registry.py

```diff
@@ -1,34 +1,23 @@
-from pathlib import Path
-from typing import Any, Callable, Dict
+# type: ignore
```

Comment:
why?

Reply:
I have to write too many "type ignores" instead 
I